searchJob/tests_2.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+from accounts.models import UserAccount
+from django.contrib.auth.models import User
+from user_notification.models import JobType, JobCity
+from.models import JobPost, UserApplication, Company
+
+
+@pytest.mark.parametrize('n,expected', [
+    (("shakedch", "APPLIED"), "<QuerySet [<UserApplication: intel sw for students>, <UserApplication: intel QA>]>"),
+    (("shakedch", None), "<QuerySet [<UserApplication: intel sw for students>, <UserApplication: intel QA>]>"),
+    (("non_relevant", "APPLIED"), "No results found"),
+    (("shakedch", "NOT_APPLIED"), "No results found"),
+    (("itay11", "APPLIED"), "<QuerySet [<UserApplication: intel QA>, <UserApplication: Red Hat QA>]>"),
+    (("itay11", "NOT_APPLIED"), "<QuerySet [<UserApplication: intel sw for students>]>"),
+    (("itay11", None), "<QuerySet [<UserApplication: intel sw for students>, <UserApplication: intel QA>, "
+                       "<UserApplication: Red Hat QA>]>"),
+    ((None, "NOT_APPLIED"), "<QuerySet [<UserApplication: intel sw for students>]>"),
+    ])
+@pytest.mark.django_db
+def testGetUserApplications(n, expected):
+
+    # initialize the DB with data
+    insertDataToDB()
+
+    # Check few cases
+    # Check assert (realFunctionResult, expected)
+    assert str(UserApplication.GetUserApplications(*n)) == expected
+
+    # Remove all the objects from DB
+    deleteAllobjects()
+
+
+def deleteAllobjects():
+    JobPost.objects.all().delete()
+    JobType.objects.all().delete()
+    JobCity.objects.all().delete()
+    Company.objects.all().delete()
+    User.objects.all().delete()
+    UserAccount.objects.all().delete()
+    UserApplication.objects.all().delete()
```

Comment:
if there is other data in DB wouldn't this delete everything? I think it's better to delete specifically the objects you create (wdyt? @bardielle @benipeled)

Reply:
Agree with you
Please open an issue or fix it as part of that PR

Reply:
Hi, I tried it with manual data insertion Through the ui, and it didn't delete all the data. 
i will try again and try to fix that.
ty

Reply:
@itaiv95 any update?

Reply:
Hi, yes I checked that and it doesn’t delete all the data- only the test data

Reply:
@itaiv95 how does it delete only the test data if you do all().delete without any filter?

Reply:
Of course it's not- when you run a test it creates a local env that simulator a DB and not running against on the real DB..
Still - you need to remove explicit rows...

Reply:
@itaiv95 ^

Reply:
Ok, thank you i will fix and update it today
Thank you both!

Reply:
done 
@guy9050 @bardielle